src/main/java/net/sf/jabref/model/database/EntrySorter.java

```diff
@@ -90,8 +90,10 @@ public void databaseChanged(DatabaseChangeEvent e) {
             int pos;
             switch (e.getType()) {
             case ADDED_ENTRY:
-                pos = -Collections.binarySearch(set, e.getEntry(), comp) - 1;
-                set.add(pos, e.getEntry());
+                pos = Collections.binarySearch(set, e.getEntry(), comp);
```

Comment:
I don't understand this change. Previously, the coded added always an entry, now only if it would be position < 0 ?